### PR TITLE
[MC-2064] Add Proxy and SpikyProxy Domain for Android and iOS platforms

### PR DESCRIPTION
**Pull request by AlokKumar-CT**
Add LaunchWithCredentialsForProxyServer in Plugin, Clevertap and platformBindings

### CODE REVIEW

File: CleverTap/Plugins/iOS/CleverTapUnityManager.mm [hunk 1]
```diff
@@ -74,8 +74,17 @@ + (void)launchWithAccountID:(NSString*)accountID andToken:(NSString *)token {
...
     [CleverTap setCredentialsWithAccountID:accountID token:token region:region];
+    [self performCommonLaunchTasks];
+}
+
++ (void)launchWithAccountID:(NSString *)accountID token:(NSString *)token proxyDomain:(NSString *)proxyDomain spikyProxyDomain:(NSString *)spikyProxyDomain {
+    [CleverTap setCredentialsWithAccountID:accountID token:token proxyDomain:proxyDomain spikyProxyDomain:spikyProxyDomain];
+    [self performCommonLaunchTasks];
+}
+
++ (void)performCommonLaunchTasks {
```
Comment (nzagorchev) [inline CleverTap/Plugins/iOS/CleverTapUnityManager.mm:87]: Please rename this method. Use launch or something descriptive.

**Review by nzagorchev — CHANGES_REQUESTED** [review]

File: CleverTap/Plugins/iOS/CleverTapUnityManager.mm (hunk 1 above)
Comment (AlokKumar-CT) [reply CleverTap/Plugins/iOS/CleverTapUnityManager.mm:87]: Renamed it to initializeAndNotifyLaunch

File: CleverTap/Plugins/iOS/CleverTapBinding.m [hunk 2]
```diff
@@ -135,6 +135,10 @@ void CleverTap_launchWithCredentialsForRegion(const char* accountID, const char*
     [CleverTapUnityManager launchWithAccountID:clevertap_stringToNSString(accountID) token:clevertap_stringToNSString(token) region:clevertap_stringToNSString(region)];
 }
 
+void CleverTap_launchWithCredentialsForProxyServer(const char* accountID, const char* token, const char* proxyDomain, const char* spikyProxyDomain) {
```
Comment (nzagorchev) [inline CleverTap/Plugins/iOS/CleverTapBinding.m:138]: I believe this should call the `CleverTapUnityManager launchWithAccountID:::::` overload with the `proxyDomain` and `spikyProxyDomain`.

**Review by nzagorchev — CHANGES_REQUESTED** [review]

**Review by nzagorchev — CHANGES_REQUESTED** [review]
Please add the new launch method to the `IOSDllImport` class.

**Review by nzagorchev — APPROVED** [review]

**Pull request merged**